### PR TITLE
[feat] - add new instance types for dev spot node, updated NVME creation script to include more types

### PR DESCRIPTION
**Pull request by mur-me**

### CODE REVIEW

**Comment by mur-me** [comment]
Will apply this on Monday, now I wanna chill.

**Review by ArseniiPetrovich — DISMISSED** [review]
Niiiiiiiiice!

File: modules/eks_nodegroup/bootstrap/early-customizations/test_nvme.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/bin/env bash
+
+is_nvme() {
+  NMVE_INSTANCES=(c6gd g4ad g4dn g5 i3 i3en i4i i4i im4gn is4gen m5d m5dn m6gd m6id r5ad r5dn r6gd r6id)
+  # Retrieve instance metadata from IMDSv2
+#   token=$(curl -X PUT "http://169.254.169.254/latest/api/token" -H "X-aws-ec2-metadata-token-ttl-seconds: 21600")
+#   current_type=$(curl -H "X-aws-ec2-metadata-token: ${token}" -v http://169.254.169.254/latest/meta-data/instance-type)
+  # Filter instance type. The goal is to detect NVME instance, which we need to format and mount.
+  # Link to check NVMEs - https://instances.vantage.sh/?region=ap-northeast-1
+  current_type="i3.2xlarge"
```
Comment (witnsby) [inline modules/eks_nodegroup/bootstrap/early-customizations/test_nvme.sh:10]: Is it important to keep `current_type="i3.2xlarge"` insdead `current_type=""` ?

File: modules/eks_nodegroup/bootstrap/early-customizations/test_nvme.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#!/bin/env bash
+
+is_nvme() {
```
Comment (witnsby) [inline modules/eks_nodegroup/bootstrap/early-customizations/test_nvme.sh:3]: It's a minor change but I know in the shell script we have a lot about nvme. What do you think about making the function name a little bit more understandable?  for instance `is_instance_type_nvme`

**Review by witnsby — DISMISSED** [review]

**Review by ArseniiPetrovich — APPROVED** [review]

**Comment by mur-me** [comment]
This one was already applied to mainnet by @dumikau and @lazavikmaria

**Review by dumikau — APPROVED** [review]

**Pull request merged**